docs/topics/proc_creating-an-application.adoc

```diff
@@ -0,0 +1,38 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+:_mod-docs-content-type: PROCEDURE
+
+[id="creating-an-application_{context}"]
+= Creating an application by using MTA
+
+[role="_abstract"]
+You can create a new application by using the {ProductName} {ProductShortName} user interface.
+
+.Prerequisites
+
+* You are logged in to an {ProductShortName} server.
+
+.Procedure
+
+. In the {ProductShortName} user interface, select the *Migration* working mode.
+. Click *Application Inventory* in the left menu bar.
+. Click *Create new*. The *New application* dialog appears.
+. In the form, enter the following information:
+.. *Name*: A unique name for the new application.
+.. *Description*: A short description of the application (optional).
+.. *Business service*: A purpose of the application (optional).
+.. *Tags*: Software tags that characterize the application.
+.. *Owner*: A registered software owner from the drop-down list (optional).
+.. *Contributors*: Contributors from the drop-down list (optional).
+.. *Comments*: Relevant comments on the migrated application.
+. Click *Source code* and enter the following fields:
+.. *Repository type*: *Git* or *Subversion*.
+.. *Source Repository*: A URL of the repository where the software code is saved.
+.. *Branch*: An application code branch in the repository.
+.. *Root path*: [get info] (optional/mandatory).
+. Click *Binary* and enter the following fields:
+.. *Group*: [get info] (optional/mandatory).
+.. *Artifact*: [get info] (optional/mandatory).
+.. *Version*: A software version of the application.
+.. *Packaging*: [get info] (optional/mandatory).
```

Comment:
"The packaging for the application artifact: JAR/WAR/EAR"

Reply:
Ack